src/modules/commander/Commander.hpp

```diff
@@ -285,6 +270,7 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 	uORB::Subscription					_vehicle_acceleration_sub{ORB_ID(vehicle_acceleration)};
 
 	uORB::SubscriptionData<airspeed_s>			_airspeed_sub{ORB_ID(airspeed)};
+	uORB::SubscriptionData<airspeed_validated_s>			_airspeed_validated_sub{ORB_ID(airspeed_validated)};
```

Comment:
Is any of this still being used in commander?

_airspeed_sub, _airspeed_validated_sub, _airspeed_fault_declared, _time_airspeed_fault_declared